kcidb/db/__init__.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, database):
                                   driver
         """
         assert isinstance(database, str)
+        self.database = database
```

Comment:
Normally, what you'd do in this situation, is to cherry-pick the commit you need into this PR's branch, and then add your changes in a separate commit on top, mentioning in the PR description, that it requires that other PR to get merged first. Then you drop the cherry-picked commit after that PR is merged by rebasing your branch on latest `main` and pushing to the PR again.

But you can leave this as is for this PR and focus on the main change for now.

Reply:
Now that [your PR](https://github.com/kernelci/kcidb/pull/409) adding this is merged, you can rebase your PR on top of latest main, and get rid of this change in it.